app/views/data_files/show.html.haml

```diff
@@ -67,9 +71,9 @@
         - else
           - start_date = @data_file.start_time.present? ? @data_file.start_time.utc.to_s(:with_seconds) : "Unknown"
           - end_date = @data_file.end_time.present? ? @data_file.end_time.utc.to_s(:with_seconds) : "Unknown"
-
-        - render_field "Start time", start_date
-        - render_field "End time", end_date
+        - unless @data_file.is_ncml? # Can't calculate start and end time for ncml files
```

Comment:
I thought about it at the time (all those ifs and unless is so ugly..), and decided against it because of effort required. Will refactor if time permits!